datadog-service/0.15.0/artifacthub-pkg.yml

```diff
@@ -9,22 +9,23 @@ homeURL: https://keptn.sh/docs/integrations/
 keywords:
 - keptn
 - sandbox
+- observability
```

Comment:
Added keywords to fix https://github.com/keptn-sandbox/datadog-service/issues/45

Reply:
^ probably should be a part of another PR but I thought it was a small change and could go into this PR. If the reviewers think I should create another PR, please let me know and I will create one.